### ISSUE TITLE
Update text for prospective registration plot

### ISSUE THREAD

**Issue by delwen** [quest-bih/clinical-dashboard#19]
For the prospective registration metric, we will include more recent trials using a dataset that Nico previously extracted from the AACT (15.03.2019). It includes trials with a start date between 2006 – 2018 (2018 still included completely but not 2019).

TODO: update the Methods text accordingly.

**Comment by maia-sh** [comment]
Prepping the IV exclusion variables now, so we can choose to use as needed. Looking at our filters at https://github.com/quest-bih/clinical-dashboard/blob/main/prep/prep-intovalue.R#L9

I think we only need:
        iv_status,
        iv_interventional,

Right?

Disregard:

        iv_completion → since we want more recent trials

        has_german_umc_lead → since should all be via manual checks (I emailed NR just now to confirm)
       
 ## In case of dupes, exclude IV1 version
        !(is_dupe & iv_version == 1) → IV dupes are irrelevant here

**Comment by maia-sh** [comment]
While it's no problem to add the boolean flags, I'm not sure we should filter for `iv_status` since that excludes trials that would still be eligible to be inspected for prospective reg...
```
    iv_status = if_else(
      recruitment_status %in% c("Completed" , "Terminated" , "Suspended", "Unknown status"), TRUE, FALSE
    )
```

**Comment by maia-sh** [comment]
While I'm here, I can also switch start date filter (by 2018) to another boolean in case we change our mind to include trials started after 2018 (22 trials) [or less logically, with na start date (1 trial)]

**Comment by maia-sh** [comment]
Ok, I made the updates to prospective reg dataset and pushed to the branch: https://github.com/maia-sh/intovalue-data/commit/88c1103e98dd5ae54b2a6fad61d804e825f79fcd

@bgcarlisle : for the dashboard you could now update to exclude based on `start_by_2018` and `iv_interventional`. As said above, i would not exclude based on `iv_status` but open for counterarguments.

@delwen for the text, i reran the query to ctgov so use today's date for latest query. also updated in `query.log`

**Comment by delwen** [comment]
- `iv_completion` &rightarrow; agree to disregard
- `iv_interventional` &rightarrow; agree to keep
- `iv_status` &rightarrow; given that DRKS and CT.gov prospective reg data is displayed in the same plot (with toggle), it could be confusing if we apply different `iv_status` inclusion/exclusion criteria to these datasets. But I see the argument to extend beyond completed trials. Would suggest we either re-apply the `iv_status` to the updated CT.gov dataset (and lose trials) or disregard the `iv_status` filter in both datasets. Note that we probably couldn't drop this filter completely; we'd probably still need to filter out trials that are 'Not yet recruiting' and 'Withdrawn' (we'd need to check this again).
- `has_german_umc_lead` &rightarrow; let's wait for update from Nico
- duplicates: I guess there aren't any duplicate trials in this dataset?

**Comment by maia-sh** [comment]
`iv_status` → ah I forgot about that! indeed could be confusing...
no duplicated in this dataset (I check that assertion)

**Comment by maia-sh** [comment]
Changes pushed. I checked for 2006 min (all good). @bgcarlisle [here](https://github.com/maia-sh/intovalue-data/blob/ctgov-2018/data/ctgov-2018/prospective-reg-ctgov-2018-trials.csv) is the dataset with exclusions filtered so ready to use.

@delwen Here's some draft text just for content
This dataset includes trials led by a German UMC (based on manual evaluation [checking with Nico]) and registered in ClinicalTrials.gov. Trials were started between 2006 and 2018 inclusive per registry data available on 2021-10-06. In line with the primary IntoValue dataset, all are interventional trials with the eligible statuses ("Completed" , "Terminated" , "Suspended", "Unknown status").

**Comment by delwen** [comment]
Just adding that we decided to keep the `iv_status` exclusions for both the CT.gov and DRKS datasets even though we don't strictly need to only focus on completed trials for prospective registration. This means we lose ~500 CT.gov trials, but we gain parity between the plots and avoid having too many separate data sources with different criteria.

**Comment by delwen** [comment]
Nico suggested we limit the range of the DRKS prospective registration plot to start from 2006 (in line with the CT.gov plot). We'd lose 10 trials. I tend to agree. Again, parity but also because the earlier dates might cause confusion. Thoughts?

**Comment by delwen** [comment]
@bgcarlisle @maia-sh agree?

**Comment by bgcarlisle** [comment]
Yeah agreed! This should do it! https://github.com/quest-bih/clinical-dashboard/commit/9cff831baf944b1786aabf814d6e77ad6a35cde7

**Comment by delwen** [comment]
TODO: detail the inclusion/exclusion criteria for the different prospective registration datasets in more detail in the dashboard methods/tooltips.
